runtime/onert/core/src/exec/train/TrainableExecutor.cc

```diff
@@ -33,10 +33,10 @@ TrainableExecutor::TrainableExecutor(
   backend::train::TrainableBackendContexts &&backend_contexts,
   const compiler::train::TensorRegistries &tensor_regs,
   compiler::train::TrainableCodeMap &&code_map, const std::vector<ir::OperationIndex> &order,
-  const util::TracingCtx *tracing_ctx)
+  const util::TracingCtx *tracing_ctx, const ir::train::LossInfo &loss_info)
```

Comment:
(Optional) It would be better to reflect `LossInfo` to the `TrainableCodeMap` for Loss op instead of adding it as a param here.

Reply:
@ragmani 
It's a good suggestion. I will try to use `TrainableCodeMap` for `LossInfo`. I will take a closer look and update it again.

Reply:
Well, when I check the code, the `TrainableCodeMap` is a map structure that has `TrainableCodeAndInfo` for each operation. I don't think `LossInfo` is information related to each operation. Rahter, it is related to `TrainableExecutor` getting loss value. So in this case, I think the current location of `LossInfo` is good. What do you think about it?

Reply:
I think it's better to reflect the whole `LossInfo` to the kernels for Loss op if `LossInfo` is not changed. So I guess loss values can be reflected in output of Loss if reduction type is not changed.
Anyway, it's an optional suggestion.